src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -92,14 +92,28 @@ public void scrollToElement(By elementBy) {
   public void scrollToElement(WebElement element) {
     try {
       js.executeScript(
-          "var x = $(arguments[0]); " + "window.scroll(0,parseInt(x.offset().top - 50));", element);
+          "var x = $(arguments[0]);" + "window.scroll(0,parseInt(x.offset().top - 50));", element);
     } catch (WebDriverException e) {
       if (e.getMessage().contains(XSSContent.NO_JQUERY_ERROR)) {
         PageObjectLogging.log("JSError", "JQuery is not defined", false);
       }
     }
   }
 
+  public void scrollToSpecificElement(WebElement element) {
+//    JavascriptExecutor je = (JavascriptExecutor) driver;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)